examples/nixnet/CAN_SIGNAL_SINGLE_POINT_OUTPUT.py

```diff
@@ -0,0 +1,120 @@
+r""" Write Signal Data.
```

Comment:
Change file name to follow conventions for python.

Maybe `can-signal-single-point-output.py`?